graph/src/util/lfu_cache.rs

```diff
@@ -145,9 +153,9 @@ impl<K: Clone + Ord + Eq + Hash + Debug, V: CacheWeight + Default> LfuCache<K, V
         self.queue.len()
     }
 
-    pub fn evict(&mut self, max_weight: usize) {
+    pub fn evict(&mut self, max_weight: usize) -> Option<(usize, usize, usize)> {
```

Comment:
It would be good to comment on what this return value is.

Reply:
Done